app/assets/javascripts/components/bulkUploadImport.jsx

```diff
@@ -192,28 +199,26 @@ class BulkUploadImport extends React.Component {
       });
       that.initializeSelectTag()
       that.initializeSelectAll()
-    }).catch(function (error) {
+    })
+    .catch((error) => {
      that.setState({
       submitting: false,
       invalid: true,
-      errorMessage: JSON.stringify(error.response)
+      errorMessage: error.response.data.status || 'Something went wrong'
```

Comment:
what does the status tell us from the server side?

Reply:
it returns the specific error object returned from the server - 'No samples imported under bulk_path'.  @yunfangjuan